daemon/emer-boot-id-provider.c

```diff
@@ -105,18 +104,16 @@ emer_boot_id_provider_get_property (GObject    *object,
       break;
 
     default:
-      G_OBJECT_WARN_INVALID_PROPERTY_ID (object, property_id, pspec);
+      G_OBJECT_WARN_INVALID_PROPERTY_ID (self, property_id, pspec);
```

Comment:
Just looked upstream and apparently GLib itself always does this the way it was written originally. Does dropping [this commit](https://github.com/endlessm/eos-event-recorder-daemon/commit/e52f8798b630f7c9d170c9094972a80bde9cddab) entirely seem like a reasonable way of addressing this comment?